### DISCUSSION TITLE
Messenger with Doctrine transport not consuming some messages

### DISCUSSION THREAD

**Discussion by ziat1988** [symfony/symfony#43614] [Q&A] [answered]
**Symfony version(s) affected**: 4.4

**Description**
My task is sending a bulk emails. And what i do is in the loop, each email will be sent as a message.
When I run the command "php -d memory_limit=-1 bin/console messenger:consume async -vv", for some reasons, and sometimes  there are still some records in my database.
I have tried stop workers and re run command but it only works for the news messages. For the old ones, they are still there.
They are however with queue name 'defaut' .

**How to reproduce**
framework:
    messenger:
        failure_transport: failed

        transports:
            # https://symfony.com/doc/current/messenger.html#transport-configuration
            async: '%env(MESSENGER_TRANSPORT_DSN)%'
            failed: 'doctrine://default?queue_name=failed'
            # sync: 'sync://'

        routing:
            # Route your messages to the transports
            'App\Message\SendNewsletterMessage': async

So, is it a bug?
Can I have a solution to force consume these messages still in database? 
Thanks a lot

**Comment by welcoMattic** [comment, marked as answer]
Hello @ziat1988,

What is the value of `MESSENGER_TRANSPORT_DSN` env var? 
Can you share the logs of your command `php -d memory_limit=-1 bin/console messenger:consume async -vv`, normally you can see message processing on it.

**Reply by ziat1988** [reply]
Hi @welcoMattic ,
I found out that it's not a bug but a probleme timezone. 
Thanks for your time.
